README.md

```diff
@@ -60,11 +59,11 @@ We next calculate each signature's activity for each sample in the dataset
 with the function `calculate_activity()`.
 
 ####Active signature detection
-Now we want to identify signatures whose activities strongly vary with sample
-phenotypes, such as signatures whose activities are being significantly different
-in a comparison of interest. We recommend using the more robust
-[limma](https://bioconductor.org/packages/release/bioc/html/limma.html) approach
-to test differential activation when sample size is small.
+We next identify signatures whose activities strongly vary with treatments,
+such as signatures whose activities are significantly differentially expressed
```

Comment:
Do we say "activities are differentially expressed"? I feel "activity" is not something that can be "expressed".